### PR TITLE
improve(affiliates): week 6 developer/deployer mining payouts

### LINKED ISSUE #2333: deployer rewards 2020-12-14 23:00:00 to 2020-12-21 23:00:00
**Why**

EMP deployer rewards for 6th week from 2020-12-14 23:00:00 to 2020-12-21 23:00:00

Contract | EMP Address | Payout Address
-- | -- | --
uUSDwETH-DEC | 0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
uUSDrBTC-DEC | 0xaBBee9fC7a882499162323EEB7BF6614193312e3 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
pxUSD_MAR2021 | 0x3a93E863cb3adc5910E6cea4d51f132E8666654F | 0x53911776641d6dF38B88b9eF27f920c617E3Cb5e
uGAS-JAN21 | 0x516f595978D87B67401DaB7AfD8555c3d28a3Af4 | 0xdD395050aC923466D3Fa97D41739a4ab6b49E9F5
Mario Cash-JAN-2021 | 0x4aa79c00240a2094ff3fa6cf7c67f521f32d84a2 | 0x7eb1e51351380E67F641B84545bf41eCdc10298E 
BCHNrBTC-JAN-2021 | 0xf32219331a03d99c98adf96d43cc312353003531 | 0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David Adams <david@umaproject.org>

**Motivation**

#2333

**Summary**

week6.json
```
{
    "empWhitelist": [
      ["0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0xaBBee9fC7a882499162323EEB7BF6614193312e3", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
      ["0x516f595978D87B67401DaB7AfD8555c3d28a3Af4", "0xdD395050aC923466D3Fa97D41739a4ab6b49E9F5"],
      ["0x4AA79c00240a2094Ff3fa6CF7c67f521f32D84a2", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
      ["0xf32219331A03D99C98Adf96D43cc312353003531", "0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577"]
    ],
    "startTime": 1607986800000,
    "endTime": 1608591600000,
    "totalRewards": "50000",
    "network": 1
  }
```
To run

   `node app/DeployerRewards week6.json --network=mainnet_mnemonic`

**Issue(s)**

Fixes #2333

### CODE REVIEW

File: packages/affiliates/examples/dapp-mining/README.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ The examples will output the data fields on the transaction you would need to si
 point to integrate the feature into your own front end.
 
 **[Web3 Example](./web3-tagging.js)**
+
```
Comment (daywiss) [inline packages/affiliates/examples/dapp-mining/README.md:10]: sorry snuck this edit in for the readme, spacing was not right

**Comment by smb2796** [comment]
Verified the payout addresses and they look correct. Rewards amounts also look proportionally correct.

**Review by chrismaree — APPROVED** [review]
LGTM!

**Comment by hlambur** [comment]
All payouts were distributed

**Pull request merged**